content/quickstart/nodejs/metrics.md

```diff
@@ -172,10 +173,10 @@ First, we will create the variables needed to later record our metrics.
 const stats = new Stats();
 
 // The latency in milliseconds
-const mLatencyMs = stats.createMeasureDouble("repl/latency", MeasureUnit.MS, "The latency in milliseconds per REPL loop");
+const mLatencyMs = stats.createMeasureDouble("repl_latency", MeasureUnit.MS, "The latency in milliseconds per REPL loop");
```

Comment:
Please revert these underscore changes, they just happen to mask a problem with the Node.js Prometheus exporter and don't match naming guidelines. It is the work of the exporter to translate it accordingly to an underscore via sanitization.